extension/src/Common.ts

```diff
@@ -1,23 +1,33 @@
-import * as fs from 'fs';
-import * as path from 'path';
-
-export function replaceAll(text: string, searchFor: string | RegExp, replaceValue: string): string {
-  const re = new RegExp(searchFor, 'g');
+import * as fs from "fs";
+import * as path from "path";
+
+export function replaceAll(
+  text: string,
+  searchFor: string | RegExp,
+  replaceValue: string
+): string {
```

Comment:
Are the linebreaks based on columns or number of parameters?

Reply:
If it's based on column count I would like to suggest at least 80

Reply:
does not seem to be based on columns :)

Reply:
The default (that we are using) is 80 chars. 
If you put all those params on one line they are 101 chars

I guess that if you have one line break between two params, prettier wants you to have each one param per line...

But the beauty of this is that we will not have to care, just accept the formatting :D